README.md

```diff
@@ -10,18 +10,45 @@
 
 ## Overview
 
-**Cloud Native Development** is about moving your entire development workflow to your kubernetes cluster to:
+Kubernetes has made it very easy to deploy applications to the cloud at a higher scale than ever, but the development practices have not evolved at the same speed as application deployment patterns.
 
-1. Eliminate integration issues by developing the same way code runs in production.
-1. Write code as usual, from your favorite IDE.
-1. Forget about building, pushing or deploying containers. 
+Today, most developers try to either run parts of the infrastructure locally, or just test these integrations directly in the cluster via CI jobs or the "docker build, docker push, kubectl apply" cycle. It works, but this workflow is painful and incredibly slow.
 
-Okteto helps you achieve this with a mix of kubernetes automation, file synching between your local file system and kubernetes and hot reloading of containers.
+Okteto makes this cycle a lot faster by letting you launch your development environment directly in your Kubernetes cluster.
 
-- Learn more details on [how it works](docs/how-does-it-work.md).
+## Features
+
+### Development environments on demand 
+Your development environment is defined in a [simple yaml manifest](https://okteto.com/docs/reference/manifest).
+- Run `okteto init` to inspect your project and generate your own config file 
+- Run `okteto up` to launch your development environment in seconds. 
+
+Check in your `okteto.yml` into your repo and make collaboration easier than ever. `git clone`, `okteto up` and you're ready to go.
```

Comment:
`Check` is confusing, seems like you ask for checking the content of `okteto.yml`. I would say something like "Add" or "Commit". 

`git clone`, `okteto up` and you're ready to go.` I think this is not trivial to understand. Explain that every developer would develop in a replicable development environment just typing `okteto up`.